src/main/java/com/datalogics/pdf/hsm/samples/HsmManager.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Datalogics Inc.
+ */
+
+package com.datalogics.pdf.hsm.samples;
+
+import com.adobe.pdfjt.core.credentials.Credentials;
+
+/**
+ * The basic interface for logging into a HSM machine.
+ */
+public interface HsmManager {
+
+    /**
+     * Performs a login operation to the HSM device. The login operation occurs on the given tokenLabel or the first
+     * available slot if tokenLabel is null. NOTE: tokenLabel is synonymous with partition name.
+     *
+     * @param tokenLabel- The label of the token to which to login
```

Comment:
I see a CheckStyle complaint on this line: we need a space before the hyphen. `tokenLabel- ` -> `tokenLabel - `